hail/python/hail/expr/functions.py

```diff
@@ -351,8 +351,8 @@ def switch(expr) -> 'hail.expr.builders.SwitchBuilder':
     return SwitchBuilder(expr)
 
 
-@typecheck(f=anytype, exprs=expr_any)
-def bind(f: Callable, *exprs):
+@typecheck(f=anytype, exprs=expr_any, _ctx=nullable(str))
+def bind(f: Callable, *exprs, _ctx=None):
```

Comment:
can we add a test for `bind` creating AggLets during aggs and scans?

Reply:
done